drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java

```diff
@@ -269,7 +269,9 @@ public void onOverviewSelected() {
      */
     @Override
     public void addBackgroundPage(final ScenarioGridWidget scenarioGridWidget) {
-        kieView.getMultiPage().addPage(BACKGROUND_TAB_INDEX, new PageImpl(scenarioGridWidget, ScenarioSimulationEditorConstants.INSTANCE.backgroundTabTitle()) {
+        final MultiPageEditorViewImpl editorMultiPageView = (MultiPageEditorViewImpl) kieView.getMultiPage().getView();
+        final int backgroundPageIndex = editorMultiPageView.getPageIndex(CommonConstants.INSTANCE.EditTabTitle()) + 1;
```

Comment:
We can avoid magical numbers by using ++

Reply:
@dupliaka I was not able to apply `++` operator, because it is need to use it just with non final variables. However I decided at least update formatting == I extracted some constants to variables to not exceed maximum amount of characters per line.